### PR TITLE
Refresh catchall

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

New auth refresher to be used with the catchall-operations of fedcloud. Reads credentials for each VO from different files located under a configurable path

### CODE REVIEW

File: cloud_info_provider/auth_refreshers/oidc_refresh.py [hunk 1]
```diff
@@ -5,31 +5,43 @@
 
 
 class OidcRefreshToken(auth_refreshers.BaseRefresher):
```
Comment (gwarf) [inline cloud_info_provider/auth_refreshers/oidc_refresh.py]: Would be good to have a few doc strings

File: cloud_info_provider/auth_refreshers/oidc_vo_refresh.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import os.path
+
+from cloud_info_provider.auth_refreshers import oidc_refresh
+
+
+class OidcVORefreshToken(oidc_refresh.OidcRefreshToken):
```
Comment (gwarf) [inline cloud_info_provider/auth_refreshers/oidc_vo_refresh.py]: Here also some doc would be nice (for others/future people/me)

File: cloud_info_provider/tests/test_auth_refreshers.py [hunk 3]
```diff
@@ -101,3 +102,47 @@ def test_refresh_bad_json(self, m_post):
         self.assertRaises(RefresherException,
                           self.refresher.refresh,
                           self.provider)
+
+
+class OidcRefreshVOTest(base.TestCase):
```
Comment (gwarf) [inline cloud_info_provider/tests/test_auth_refreshers.py]: Here also some simple doc may be nice

**Comment by enolfc** [comment]
Some documentation added

**Pull request merged**